src/docs/spec-formats-tsv.md

```diff
@@ -203,7 +201,6 @@ When writing the metadata block, a canonical SSSOM/TSV writer:
 * MUST serialise multi-valued slots as YAML “block sequences” ([YAML Specification §8.2.1](https://yaml.org/spec/1.2.2/#821-block-sequences)) – even when the list of values contains only one item;
 * MUST serialise scalar values in YAML “plain style” ([YAML Specification §7.3.3](https://yaml.org/spec/1.2.2/#733-plain-style)) whenever possible, otherwise in “double-quoted style” ([YAML Specification §7.3.1](https://yaml.org/spec/1.2.2/#731-double-quoted-style));
 * MUST serialise the slots in the order they appear in the [“Slots” table](MappingSet.md#slots), in the documentation for the `MappingSet` class;
-* MUST write the `curie_map` at the beginning of the block, before any other slots;
```

Comment:
I dont think we can drop this, as the element is not tagged "required" in the spec. The fact that it is written at the beginning is redundant, yes, but not the fact that it must be written at all.

Reply:
The fact that the `curie_map` must be written at all is a general requirement that applies to all SSSOM/TSV files, not only to the ”canonical” variant. And that requirement is already stated elsewhere (“Identifiers” section):

> As stated in the description of the model, all prefix names used in CURIEs MUST be declared in the `curie_map` slot of the mapping set object

which obviously implies that the `curie_map` slot itself MUST be present, even if it is not marked as “required” in the LinkML schema.

So the removed line is completely redundant.

Reply:
Yep makes sense. Thanks!